rpmio/rpmkeyring.cc

```diff
@@ -71,6 +71,20 @@ rpmKeyring rpmKeyringFree(rpmKeyring keyring)
     return NULL;
 }
 
+size_t rpmKeyringSize(rpmKeyring keyring, rpmFlags flags)
+{
+    assert(flags == 0);
```

Comment:
BTW this is a pretty antisocial way to validate flags, we don't really do this.

Are you visioning other flags besides counting only primary vs including subkeys? If it's just a boolean (zero vs nonzero) then we wouldn't even need to validate it really.

Reply:
*urgs* I thought I did the same thing previously but I guess I only returned zero there, too.

Adding those APIs in C is a real pain. Having default values and bools would be real nice...

Reply:
ok, it is a "boolean" now.